drivers/interrupt_controller/intc_exti_stm32.c

```diff
@@ -412,17 +415,18 @@ static int stm32_exti_init(const struct device *dev)
 }
 
 static struct stm32_exti_data exti_data;
-DEVICE_DEFINE(exti_stm32, STM32_EXTI_NAME, stm32_exti_init,
-	      NULL, &exti_data, NULL,
-	      PRE_KERNEL_1, CONFIG_KERNEL_INIT_PRIORITY_DEVICE,
-	      NULL);
+DEVICE_DT_DEFINE(DT_NODELABEL(exti), &stm32_exti_init,
```

Comment:
Any reason not to use `EXTI_NODE` here instead of NODELABEL?

Reply:
None except find and replace didn't catch it, will fix.